script.js

```diff
@@ -0,0 +1,18 @@
+const mobileBtn = document.getElementById('menu-bar');
+const exitBtn = document.getElementById('close-menu');
+const aboutBtn = document.getElementById('about-2');
+const PortfolioBtn = document.getElementById('portifolio-2');
+const contactBtn = document.getElementById('contact-2');
+const nav = document.querySelector('nav');
+mobileBtn.addEventListener('click', () => {
+  nav.classList.add('menu-btn');
+  
+});
+function display() {
+  nav.classList.remove('menu-btn');
+}
+
+exitBtn.addEventListener('click', display);
+aboutBtn.addEventListener('click', display);
+PortfolioBtn.addEventListener('click', display);
+contactBtn.addEventListener('click', display);
```

Comment:
- [ ] At the break point `767px` and below, your menu items are supposed to get disappeared and replaced by hamburger icon which will display the menu items when clicked. At this moment your menu items persist appearing at that break point. Kindly look into that and fix where the error is. Below is the screenshot of how your page appears at that break point.

### Your Page

![menu-breakpoint](https://user-images.githubusercontent.com/26597432/196260471-29dde868-6730-41c7-aefd-9c7bf5c85abb.png)

Reply:
Hello Qoosim please check again it works properly

Reply:
@media only screen and (max-width: 768px) {
  nav {
    display: none;
  }